tests/randomized/Makefile

```diff
@@ -3,10 +3,10 @@ TMP_RESULTS_FOLDER := $(TMP_SCENARIO_FOLDER)/.results
 DOCKER_COMPOSE_FILE := $(TMP_SCENARIO_FOLDER)/docker-compose.yml
 REGRESSIONS_FOLDER := ./regressions
 REGRESSION_NAME := "provide-regression-name"
-DOCKER_IMAGE_PREFIX := datadog/dd-trace-ci:php-randomizedtests-
-TRACER_TEST_URL := https://github.com/DataDog/dd-trace-php/releases/download/0.54.0/datadog-php-tracer-0.54.0.x86_64.tar.gz
-TRACER_DOWNLOAD_PATH := ./.tracer-versions
-TRACER_VERSION := 0.55.0
+DOCKER_IMAGE_PREFIX := datadog/dd-trace-ci:php-randomizedtests-dev-
```

Comment:
`-dev` to be removed before or right after merge. This way other  branches do not fail.